lib/hmr/JavascriptHotModuleReplacement.runtime.js

```diff
@@ -317,8 +318,9 @@ module.exports = function () {
 								}
 							}
 							for (var k = 0; k < callbacks.length; k++) {
+								var result;
 								try {
-									callbacks[k].call(null, moduleOutdatedDependencies);
+									result = callbacks[k].call(null, moduleOutdatedDependencies);
```

Comment:
`var` is hoisted to the top of the function, so writing `var result = ...` right here without having a separate `var result` before `try` should be enough?